README.md

```diff
@@ -6,7 +6,7 @@ With Azure CLI GitHub Action, you can automate your workflow by executing [Azure
 The action executes the Azure CLI Bash script on a user defined Azure CLI version. If the user does not specify a version, latest CLI version is used.
 Read more about various Azure CLI versions [here](https://github.com/Azure/azure-cli/releases).
 
-- `azcliversion` – **Optional** Example: 2.0.72, Default: latest
+- `azcliversion` – **Optional** Example: 2.0.72, Default: set to az cli version of the agent.

```

Comment:
If the agent doesn't have cli installed then will az login fail?

Reply:
yes then command for az login will not work.